src/components/Hero/HubHero/index.tsx

```diff
@@ -1,9 +1,10 @@
-import { Box, Heading, Stack, Text } from "@chakra-ui/react"
-
 import type { CommonHeroProps } from "@/lib/types"
 
 import { CallToAction } from "@/components/Hero/CallToAction"
 import { Image } from "@/components/Image"
```

Comment:
We need to use the `TwImage` export from now on and adapt the styles accordingly.